instrumentation/formatters.go

```diff
@@ -0,0 +1,139 @@
+package instrumentation
```

Comment:
I think most of the code here should be redundant since membuffers already have a stringer for all protos (containers, primitives, etc...)

Reply:
I added support for anything that confirms to `Stringer` via `Stringable(key, value)`